source/connection-monitoring-and-pooling/tests/pool-clear-schedule-run.yml

```diff
@@ -0,0 +1,46 @@
+version: 1
+style: unit
+description: Pool clear SHOULD schedule the next background thread run immediately
+poolOptions:
+  # ensure it's not involved by default
+  backgroundThreadIntervalMS: 10000
+operations:
+  - name: ready
+  - name: checkOut
+  - name: checkOut
+    label: conn
+  - name: clear
+    closeInUseConnections: true
+  - name: waitForEvent
+    event: ConnectionPoolCleared
+    count: 1
+    timeout: 1000
+  - name: waitForEvent
+    event: ConnectionClosed
+    count: 2
+    timeout: 1000
+  - name: close
+events:
+  - type: ConnectionCheckedOut
+    connectionId: 1
+    address: 42
+  - type: ConnectionCheckedOut
+    connectionId: 2
+    address: 42
+  - type: ConnectionClosed
```

Comment:
This test and others with multiple connections closed by `closeInUseConnections: true` may potentially be racy. We should either update the test to either only close a single connection or to omit the `ConnectionClosedEvents`.